source/utilities.tex

```diff
@@ -5170,12 +5170,18 @@
 
 \indexlibrary{\idxcode{any}!constructor}%
 \begin{itemdecl}
-template <class T, class... Args>
-  explicit any(in_place_type_t<T>, Args&&... args);
+template <class ValueType, class... Args>
+  explicit any(in_place_type_t<ValueType>, Args&&... args);
 \end{itemdecl}
 
 \begin{itemdescr}
 \pnum
+Let \tcode{T} be equal to \tcode{decay_t<ValueType>}.
```

Comment:
Oops, you're right, I was looking at the superseded version. Although confusingly, the final version says "Let `T` be `decay_t<ValueType>`." even for the existing text that isn't changed (which is wrong). Bah. Anyway, I changed it.